sdk/schemaregistry/schema-registry/package.json

```diff
@@ -122,6 +122,8 @@
     "rimraf": "^3.0.0",
     "rollup": "^1.16.3",
     "source-map-support": "^0.5.9",
-    "typescript": "~4.2.0"
+    "typescript": "~4.2.0",
+    "@azure-tools/test-recorder-new": "^1.0.0",
```

Comment:
do we still need the older one at line 93?

Reply:
Nope! 😁